cmd/soci-snapshotter-grpc/main.go

```diff
@@ -136,6 +136,7 @@ func main() {
 	// Snapshotter should use "github.com/containerd/containerd/log" otherwize
 	// logs are always printed as "debug" mode.
 	golog.SetOutput(log.G(ctx).WriterLevel(logrus.DebugLevel))
+	log.G(ctx).Info(fmt.Sprintf("soci-snapshotter-grpc version %s %s", version.Version, version.Revision))
```

Comment:
I think we might want to follow containerd's lead and use structured logging here rather than formatting the string into the message. 
```
Mar 10 18:40:46 ip-172-31-26-39 containerd[18641]: time="2023-03-10T18:40:46.451521146Z" level=info msg="starting containerd" revision=2456e983eb9e37e47538f59ea18f2043c9a73640 version=1.6.18
```